library/dialog.js

```diff
@@ -518,7 +518,7 @@ const dlgopen = (url, winname, width, height, forceNewWindow, title, opts) => {
         ('<iframe id="modalframe" class="w-100 h-100 modalIframe" name="%winname%" %url% frameborder=0></iframe>')
         .replace('%winname%', winname).replace('%url%', fullURL ? 'src=' + fullURL : '');
 
-    var bodyStyles = (' style="overflow-y:auto;height:%initHeight%;max-height:92vh;"')
```

Comment:
this looks very reasonable to me
@sjpadgett , just want to make sure nothing obviously wrong about this?

Reply:
don't remember why I had to put in. something to do with certain content and breakpoints. with all the restyles, appears okay to remove. bringing in.